packages/cpp-exceptions-test/meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: cpp-exceptions-test
-  version: "0.1"
+  version: pyodide.test
```

Comment:
Yeah this looks nice to me -- even if it is a slight bit odd I think it makes sense.

Reply:
Yeah, thanks for the suggestion. The best way would be removing all those testing packages, but there is no trivial way to do that I guess?

Reply:
> The best way would be removing all those testing packages, but there is no trivial way to do that I guess?

Well the package loading logic is quite subtle, and many of the behaviors covered by these tests cover bugs that normally only occur in scipy. Also, note for instance the value of `test_cpp_exceptions` in this bug: https://github.com/emscripten-core/emscripten/issues/16538. On the whole, I think they are one of the most important value parts of the test suite.

Reply:
It's sort of similar to the reason that the Python interpreter comes with ~6 testing only extension modules.

Reply:
Thanks for the explanation. Yes I also think those tests are really valuable.

Reply:
I see the intention, but I think it would be better to add a separate key for this, maybe,
```
package:
    tag: pyodide.test
```
rather than putting this information into the version which is supposed to verify PEP-386 version conventions https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#package-version We are not checking that now, but we could very well do it in the future.

Reply:
Alright, then I'll go with adding a separate key. What do you think of keeping the new key private, so that we can modify or remove it easily when we find a better way to specify those special packages?